content/02-types.md

```diff
@@ -678,7 +679,7 @@ att.income = 0;
 <!--label:types-dynamic-access-->
 #### Dynamic access
 
-`DynamicAccess` is an [abstract type](types-abstract) for working with [anonymous structures](types-anonymous-structure) that are intended to hold collections of objects by the string key. Basically `DynamicAccess` wraps `Reflect`std-reflection calls in a Map-like interface.
+`DynamicAccess` is an [abstract type](types-abstract) for working with [anonymous structures](types-anonymous-structure) that are intended to hold collections of objects by the string key. Basically, `DynamicAccess` wraps `Reflect`std-reflection calls in a Map-like interface.
```

Comment:
That std-reflection bit should be a link, i.e. ``… wraps [`Reflect`](std-reflection) calls …``

Reply:
Sure